csharp/Ice/Secure/Client/Program.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) ZeroC, Inc.
+
+// Slice module VisitorCenter in Greeter.ice maps to C# namespace VisitorCenter.
+using VisitorCenter;
+
+// Create an Ice communicator to initialize the Ice runtime. The communicator gets its properties from file
+// "config.client".
+using Ice.Communicator communicator = Ice.Util.initialize(ref args, "config.client");
```

Comment:
Should we instead show case the new APIs using `SslClientAuthenticationOptions`?

IceSSL properties are mostly for compatibility, but the native APIs should be preferred.